### PR TITLE
completely redo/check eq_eq

### PR DESCRIPTION
**Pull request by PCelestia**
will rebase

### CODE REVIEW

File: assets/minecraft/lang/eq_eq.json [hunk 1]
```diff
@@ -1,352 +1,531 @@
...
-  "commands.title.show.subtitle.multiple": "Showing new subtitle for %s ponies",
-  "commands.title.show.actionbar.multiple": "Showing new actionbar title for %s ponies",
-  "commands.title.times.multiple": "Changed title display times for %s ponies",
-  "argument.player.entities": "Only ponies may be affected by this command, but the provided selector includes entities",
-  "commands.bossbar.set.players.unchanged": "Nothing changed. Those ponies are already on the bossbar with nobody to add or remove",
-  "clear.failed.multiple": "No items were found on %s ponies",
-
-  "death.attack.wither": "%1$s devolved into chaos",
-  "death.attack.wither.player": "%1$s devolved into chaos whilst fighting %2$s"
+   "language.name": "Modern Equish",
```
Comment (Sollace) [inline assets/minecraft/lang/eq_eq.json:2]: Please change your editor to use 2 spaces for indentation (as the original used that).

Current diff says you changed every line so it's hard to tell what the _actual_ diff should be.

File: assets/minecraft/lang/eq_eq_original.json [hunk 2]
```diff
@@ -0,0 +1,352 @@
+{
```
Comment (Sollace) [inline assets/minecraft/lang/eq_eq_original.json:1]: Why are you including multiple copies of the eq lang? Did you fix #90 ? It renders as a new file so it's impossible to check.

File: assets/minecraft/lang/eq_eq.json (hunk 1 above)
Comment (PCelestia) [reply assets/minecraft/lang/eq_eq.json:2]: sorry about that, will do

File: assets/minecraft/lang/eq_eq_original.json (hunk 2 above)
Comment (PCelestia) [reply assets/minecraft/lang/eq_eq_original.json:1]: multiple copies is temporary, will be removed

**Comment by Sollace** [comment]
I suggest keeping the order as it was before. It was grouped by class (item, block, entity, etc) as languages generally are in vanilla.

**Comment by PCelestia** [comment]
the old ordering was a mess
ill compare it to the eng langfile

**Comment by PCelestia** [comment]
i guess merge for now, if need more work on it we can open another pr

**Pull request merged**